src/components/DataTable/TableBatchActions.js

```diff
@@ -0,0 +1,45 @@
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const TableBatchActions = ({
+  className,
+  children,
+  showBatchActions,
+  totalSelected,
+  handleClick,
+  ...other
+}) => {
+  const batchActionsClasses = cx(
+    {
+      'bx--batch-actions': true,
+      'bx--batch-actions--active': showBatchActions,
+    },
+    className
+  );
+
+  return (
+    <div className={batchActionsClasses} {...other}>
+      {children}
+      <div className="bx--batch-summary">
+        <p className="bx--batch-summary__para">
+          <span>{totalSelected}</span>{' '}
+          {totalSelected > 1 ? 'items selected' : 'item selected'}
```

Comment:
Could this be made into a prop? It'd be nice to have it for internationalization.

Reply:
Good call!